apps/stateRuleEngine/stateRuleEngine.hpp

```diff
@@ -225,6 +225,9 @@ int stateRuleEngine::appStartup()
             }
 
             pcf::IndiElement elem = pcf::IndiElement(it->first, pcf::IndiElement::Off);
+            if (it->second->message() != "") {
```

Comment:
I know that libMagAOX was checking "", but I've since figured out that libcommon isn't doing anything different with defaults.  it should be fine to just add every message.